server/src/controllers/productController.js

```diff
@@ -1,56 +1,48 @@
-class productController {
-  products = [{
-    productId: 1,
-    productName: 'Item 1 ',
-    price: 19.99
-  },
-  {
-    productId: 2,
-    productName: 'Item2',
-    price: 25.99
-  }];
+import { v4 as uuidv4 } from 'uuid';
+class ProductController {
+  products = [
+    {
+      productId: '2fbf70b4-2b67-448c-97e1-9a33dcd8de27',
+      productName: 'Item 12',
+      productPrice: 19.99,
+    },
+    {
+      productId: '4bb7ca3a-070e-4be1-a4a1-d63c79fbce65',
+      productName: 'Item2',
+      productPrice: 25.99,
+    },
+  ];
 
   getAllProducts = (req, res) => {
-    res.status(200).json(this.products)
-  }
+    res.status(200).json(this.products);
+  };
 
   getProdById = (req, res) => {
```

Comment:
What do you think about changing `getProdById` to `getProductById`? In my opinion, it would be better, because right now we have `prod` and `product` words.